wgpu-hal/src/lib.rs

```diff
@@ -839,6 +839,12 @@ pub struct NagaShader {
     pub info: naga::valid::ModuleInfo,
 }
 
+/// Shader input.
+pub enum ShaderInput<'a> {
+    NagaShader(NagaShader),
```

Comment:
since the enum already has `Shader` in it, can we drop the word from the variants?

Reply:
Sure!